src/main/java/com/shoes/ordering/system/domain/common/event/DomainEvent.java

```diff
@@ -0,0 +1,5 @@
+package com.shoes.ordering.system.domain.common.event;
+
+// 해당 제네릭 타입은, 해당 이벤트를 발생시킬 엔터티의 유형이다.
+public interface DomainEvent <T>{
```

Comment:
요기 띄어쓰기를 살짝 넣어주면 좋을 것 같아요~~ㅎㅎㅎ
```suggestion
public interface DomainEvent <T> {
```

요기 안에 메서드가 없는데 어떻게 사용하는 인터페이스인가요?

Reply:
앗 ~! 이걸 이제 봤네욤! 해당 인터페이스는 Marker 클래스입니다! 정의된 기능은 없지만, 이건 DomainEvent 로 사용해! 라고 좀 더 가시적으로 표현이 되는 것 같아 사용했습니다~!

띄어쓰기는 미처 확인 못했네요..! 해당 부분 바로 슈슝 수정해야겠어요 ~!!

Reply:
하나의 명시적인 타입으로서의 기능을 하기위해 만들어진 인터페이스군요!
완전 좋은데요?? 찾아보니 마크인터페이스 패턴이라는 것도 있네요!!
배워갑니당 ~~~